### ISSUE TITLE
Pact Installer, Md5 checksum not matching...

### ISSUE THREAD

**Issue by dlgoodchild** [babun/babun#257]
There seems to be a problem with the md5 check for new packages. It's not obtaining a correct 32-char length md5 hash for the $digest value (I commented out the exit for now to move onward)....here's some sample output:

````
Installing python-setuptools
Found package python-setuptools
--2015-03-24 11:06:15--  http://mirrors.kernel.org/sourceware/cygwin//x86/release/python-setuptools/python-setuptools-0.6.34-1.tar.bz2
Resolving mirrors.kernel.org (mirrors.kernel.org)... 149.20.37.36, 198.145.20.143, 2001:4f8:4:6f:0:1994:3:14, ...
Connecting to mirrors.kernel.org (mirrors.kernel.org)|149.20.37.36|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 509812 (498K) [application/octet-stream]
Saving to: ‘python-setuptools-0.6.34-1.tar.bz2’

100%[==========================================================>] 509,812      213KB/s   in 2.3s

2015-03-24 11:06:18 (213 KB/s) - ‘python-setuptools-0.6.34-1.tar.bz2’ saved [509812/509812]

MD5 sum did not match, exiting e7effad5438bf415f22b71f9d65a70f65141aea7a9820e1fd53fa551e7188463f81e634f4bb30f5c5a5be5c0ae91cabddff8e471daafe16f8ae44da864872ea9 not equal to 697ce133990ef908b20a3c8f1fbb9ff0
Unpacking...
Package python-setuptools requires the following packages, installing:
python
Package python is already installed, skipping
Package python-setuptools installed
````

As you can see I modified the echo to show both "md5"'s

**Comment by tombujok** [comment]
Thanks, could you / somebody look into that? We are just baking the newest release - it would be great to include a fix for that too...

**Comment by rossdavidsmith** [comment]
The longer hash is a sha512 (replacing md5sum with sha512 seems to fix things).

**Comment by tombujok** [comment]
Pull request please :+1:

**Comment by rossdavidsmith** [comment]
Submitted a pull request (#258).

It's my very first pull request, so apologies if I messed anything up!

**Comment by zeludo** [comment]
Could you please tell when the next release fixing this issue is planned ? Thx

**Comment by harijoe** [comment]
+1 zeludo
or giving a way to compile babun from the sources, because at the moment the compiled version available on the website is the buggy one.

**Comment by tombujok** [comment]
We know about the problems - the final release candidate should be ready
today. New version should be out soon afterwards.

**Comment by tombujok** [comment]
Fixed in 1.1.0. 
Release status: #266
